benchmarks/build.xml

```diff
@@ -3,10 +3,14 @@
 <project basedir="." default="run" name="benchmarks">
 	<import file="../build-common.xml" />
 
-	<target name="build-sample-sql">
+	<property name="build.dir" value="build" />
+
+	<target depends="clean" name="build-sample-sql">
+		<property name="sample.sql.output.dir" value="${basedir}/${build.dir}" />
```

Comment:
@slnn Please remove this, make benchmark project copy the output to its target dir.